src/main/java/net/glowstone/net/GlowSession.java

```diff
@@ -375,12 +375,14 @@ public void setPlayer(PlayerProfile profile) {
     }
 
     public ProtocolType getProtocolType() {
-        if (version == GlowServer.PROTOCOL_VERSION)
-            return ProtocolType.PLAY;
-        if (version == GlowServer.LEGACY_PROTOCOL_VERSION)
-            return ProtocolType.PLAY_107;
-        if (version == GlowServer.COMPATIBLE_PROTOCOL_VERSION)
-            return ProtocolType.PLAY_109;
+        switch (version) {
```

Comment:
@Paulomart So would you prefer `LEGACY_PROTOCOL_1_9` and `LEGACY_PROTOCOL_1_9_2` as a replacement for `LEGACY_PROTOCOL_VERSION` and `COMPATIBLE_PROTOCOL_VERSION`?